docs/product-business-resource-planner-guide/src/main/asciidoc/optimizer-REST-API-configuration-ref.adoc

```diff
@@ -0,0 +1,419 @@
+  <solver-config-file>optacloud/optacloud/cloudSolverConfig.solver.xml</solver-config-file>
+  <status>NOT_SOLVING</status>
+  <score scoreClass="org.optaplanner.core.api.score.buildin.hardsoft.HardSoftScore">0hard/-5460soft</score>
+  <best-solution class="optacloud.optacloud.CloudSolution">
+    <computerList>
+      <optacloud.optacloud.Computer>
+        <cpuPower>24</cpuPower>
+        <memory>96</memory>
+        <networkBandwidth>16</networkBandwidth>
+        <cost>4800</cost>
+      </optacloud.optacloud.Computer>
+      <optacloud.optacloud.Computer>
+        <cpuPower>6</cpuPower>
+        <memory>4</memory>
+        <networkBandwidth>6</networkBandwidth>
+        <cost>660</cost>
+      </optacloud.optacloud.Computer>
+    </computerList>
+    <processList>
+      <optacloud.optacloud.Process>
+        <requiredCpuPower>1</requiredCpuPower>
+        <requiredMemory>1</requiredMemory>
+        <requiredNetworkBandwidth>1</requiredNetworkBandwidth>
+        <computer reference="../../../computerList/optacloud.optacloud.Computer[2]"/>
+      </optacloud.optacloud.Process>
+      <optacloud.optacloud.Process>
+        <requiredCpuPower>3</requiredCpuPower>
+        <requiredMemory>6</requiredMemory>
+        <requiredNetworkBandwidth>1</requiredNetworkBandwidth>
+        <computer reference="../../../computerList/optacloud.optacloud.Computer"/>
+      </optacloud.optacloud.Process>
+      <optacloud.optacloud.Process>
+        <requiredCpuPower>1</requiredCpuPower>
+        <requiredMemory>1</requiredMemory>
+        <requiredNetworkBandwidth>3</requiredNetworkBandwidth>
+        <computer reference="../../../computerList/optacloud.optacloud.Computer"/>
+      </optacloud.optacloud.Process>
+      <optacloud.optacloud.Process>
+        <requiredCpuPower>1</requiredCpuPower>
+        <requiredMemory>2</requiredMemory>
+        <requiredNetworkBandwidth>11</requiredNetworkBandwidth>
+        <computer reference="../../../computerList/optacloud.optacloud.Computer"/>
+      </optacloud.optacloud.Process>
+      <optacloud.optacloud.Process>
+        <requiredCpuPower>1</requiredCpuPower>
+        <requiredMemory>1</requiredMemory>
+        <requiredNetworkBandwidth>1</requiredNetworkBandwidth>
+        <computer reference="../../../computerList/optacloud.optacloud.Computer"/>
+      </optacloud.optacloud.Process>
+      <optacloud.optacloud.Process>
+        <requiredCpuPower>1</requiredCpuPower>
+        <requiredMemory>1</requiredMemory>
+        <requiredNetworkBandwidth>5</requiredNetworkBandwidth>
+        <computer reference="../../../computerList/optacloud.optacloud.Computer[2]"/>
+      </optacloud.optacloud.Process>
+    </processList>
+    <score>0hard/-5460soft</score>
+  </best-solution>
+</solver-instance>
+----
+====
+
+== [POST] /containers/{containerId}/solvers/{solverId}/problemfactchanges
+Real-time planning feature.
+Submits one or multiple ProblemFactChanges to update the dataset the solver currently optimizes.
```

Comment:
This is a complex use case, so one that would really benefit from being documented.

Reply:
I've crated a JIRA for this: https://issues.jboss.org/browse/BXMSDOC-2192.